little-books/src/components/BlogDetail/BlogDetail.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import { BlogType } from "../../types/blog.type";
+import Button from "../Button/Button";
+import "./BlogDetail.scss";
+import { useAppDispatch } from "../../hooks";
+import { updateBlog } from "../../redux/blogs/blogSlice";
+import FallbackImage from "../FallbackImage/FallbackImage";
+
+interface BlogDetailProps {
+	item: BlogType;
+}
+
+const BlogDetail: React.FC<BlogDetailProps> = ({ item }) => {
+	const dispatch = useAppDispatch();
+
+	const [isEdit, setIsEdit] = useState(false);
+	const [formObj, setFormObj] = useState<BlogType>({ ...item });
+
+	useEffect(() => {
+		setFormObj({ ...item }); // Update formObj when item changes
+	}, [item]);
```

Comment:
Keep useEffect at the last (just before return ())

Reply:
resolved